pytest_httpserver/httpserver.py

```diff
@@ -442,15 +442,15 @@ def respond(self, request: Request) -> Response:
         """
         Calls the request handler registered for this object.
 
-        If no request handler was specified previously, it raises
+        If no response was specified previously, it raises
```

Comment:
I'm sorry but I don't see the purpose of this change (and the change below in line 453). Could you please explain why you think that it is a better text?

My clarification:
This method is called when a new request arrives to the server, and it checks that if there's a "request handler" registered for it. If yes, it will delegate the handling to it, if not, it will raise an error.
Here, we cannot speak about response, as it is not yet available as the request handler (if there's any) hasn't been called yet.

Reply:
This method is inside the request handler - so a request handler was found. The case that no request handler was found is further done in this file.

In the example from the readme:
```
def test_my_client(httpserver): # httpserver is a pytest fixture which starts the server
    # set up the server to serve /foobar with the json
    httpserver.expect_request("/foobar").respond_with_json({"foo": "bar"})
    # check that the request is served
    assert requests.get(httpserver.url_for("/foobar")).json() == {'foo': 'bar'}
```

Replacing:

```
    httpserver.expect_request("/foobar").respond_with_json({"foo": "bar"})
```

with:
```
    httpserver.expect_request("/foobar")
```

would trigger the error. I think it would be great if it could be a different exception from having no`expect_request("/foobar")` at all.

Reply:
Ok, I think I'm on good path in understanding. :)

Your issue is that there's very similar text in the exception and in the response body?
Both says something like "No handler found for request".